uwblueprint/planet-read
# Add story test database changes

## Notion ticket link



[Final Story Test Architecture](https://www.notion.so/uwblueprintexecs/Final-Story-Test-Architecture-e6b2a15d9c9d4a18861da63b31ea0ad5)



## Implementation description

- Added is_test col to story tables
- Added is_test, test_grade, test_result, test_feedback columns to story_translation_all
- Created story_translation_tests view 
- Added TEST_CORRECT, TEST_INCORRECT, TEST_PARTIALLY_CORRECT enums for status



## Steps to test

1. docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db downgrade"    
2. docker exec -it planet-read_py-backend_1 /bin/bash -c "flask db upgrade"       
3. docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data"
4. open mysql cli
5. check that existing tables have correct columns (`show columns in stories;`, `show columns in story_translations_all;`,  )
6. check that correct views were created 



## What should reviewers focus on?

- Does it work?

## Checklist
